modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/util/ObjectEntryVariablesUtil.java

```diff
@@ -71,7 +74,7 @@ public static Map<String, Object> getActionVariables(
 			).put(
 				"originalObjectEntry",
```

Comment:
@p-albuquerque are we going to keep the keys `"objectEntry",` and `"originalObjectEntry",` even if the object is system object?

Reply:
I had been consider it, but if I change the key "originalObjectEntry" to use the suffix, I will need to take care of suffix in all the places that use DDMExpression. I think this will add more complexity. If you see some way to do that in a simple way, ping me

Reply:
Unless I use the index of Map position to get the value of this key.

Reply:
I was thinking of renaming it to something more generic like, for both cases (system and custom). Something like: "baseModel" and "originalBaseModel"

Reply:
Copy that!